bin/generate

```diff
@@ -151,13 +151,15 @@ function generate_hub_commit() {
   CUSTOM_NAMESPACES_FILENAME="/app/lookml-generator/custom-namespaces.yaml"
   GENERATED_SQL_URI="https://github.com/mozilla/bigquery-etl/archive/generated-sql.tar.gz"
   APP_LISTINGS_URI="https://probeinfo.telemetry.mozilla.org/v2/glean/app-listings"
+  NAMESPACE_GCS_BUCKET="wlach-test-gcs"
```

Comment:
This will obviously need to be changed before we land. @haroldwoo I guess it would be up to you to set this bucket up and make sure the container has write permissions to it (and that we can read from it via an unauthenticated https session).